AnatomyOfMatplotlib-Part3-HowToSpeakMPL.ipynb

```diff
@@ -102,17 +102,17 @@
     "# Markers\n",
     "[Markers](http://matplotlib.org/api/markers_api.html) are commonly used in [`plot()`](http://matplotlib.org/api/pyplot_api.html#matplotlib.pyplot.plot) and [`scatter()`](http://matplotlib.org/api/pyplot_api.html#matplotlib.pyplot.scatter) plots, but also show up elsewhere. There is a wide set of markers available, and custom markers can even be specified.\n",
     "\n",
-    "marker     |  description  ||marker    |  description  ||marker    |  description  ||marker    |  description  \n",
-    ":----------|:--------------||:---------|:--------------||:---------|:--------------||:---------|:--------------\n",
-    "\".\"        |  point        ||\"+\"       |  plus         ||\",\"       |  pixel        ||\"x\"       |  cross\n",
-    "\"o\"        |  circle       ||\"D\"       |  diamond      ||\"d\"       |  thin_diamond ||          |\n",
-    "\"8\"        |  octagon      ||\"s\"       |  square       ||\"p\"       |  pentagon     ||\"\\*\"      |  star\n",
-    "\"&#124;\"   |  vertical line||\"\\_\"      | horizontal line ||\"h\"     |  hexagon1     ||\"H\"       |  hexagon2\n",
-    "0          |  tickleft     ||4         |  caretleft    ||\"<\"       | triangle_left ||\"3\"       |  tri_left\n",
-    "1          |  tickright    ||5         |  caretright   ||\">\"       | triangle_right||\"4\"       |  tri_right\n",
-    "2          |  tickup       ||6         |  caretup      ||\"^\"       | triangle_up   ||\"2\"       |  tri_up\n",
-    "3          |  tickdown     ||7         |  caretdown    ||\"v\"       | triangle_down ||\"1\"       |  tri_down\n",
-    "\"None\"     |  nothing      ||`None`    |  default       ||\" \"       |  nothing      ||\"\"        |  nothing"
+    "marker     |  description  | marker    |  description    |marker    |  description  | marker    |  description  \n",
+    ":----------|:--------------| :---------|:--------------  |:---------|:--------------| :---------|:--------------\n",
```

Comment:
There is now a space between `|` and `:`. I forget if that makes a difference or not.